scripts/shared/judgeJSFilesVersion.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+const minimatch = require('minimatch');
+const {es5Path} = require('./esPath');
+
+module.exports = function(jsFiles) {
```

Comment:
If this is only used in `linc` task let’s just move it and inline it there. I wouldn’t worry too much about having a test for this file.

Reply:
ok